### DISCUSSION TITLE
设置table的滚动条的位置

### DISCUSSION THREAD

**Discussion by storageChu** [ant-design/ant-design#32076] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/ecj4g?file=/index.js)

### Steps to reproduce
如实例中的table，想要加载完 让横向滚动条位置在右侧 或者 scrollleft 100px该如何实现？

### What is expected?
可设置滚动条的位置

### What is actually happening?
暂不知如何设置滚动条的位置

| Environment | Info |
|---|---|
| antd | 4.16.13 |
| React | 16.13..1 |
| System | windows10 |
| Browser | chrome |

**Comment by SlahserZ** [comment]
```javascript
const el = document.getElementsByClassName('ant-table-body')[0];
el.scrollTo({ left: 300 });
```
